mingw-w64-cross-compiler-rt/PKGBUILD

```diff
@@ -0,0 +1,68 @@
+# Maintainer: Mateusz Mikuła <mati865@gmail.com>
+
+_realname=compiler-rt
+_mingw_suff=mingw-w64-cross
+pkgname=("${_mingw_suff}-${_realname}")
+pkgver=11.0.0
+pkgrel=1
+pkgdesc="Cross Clang for the MinGW-w64"
+arch=('i686' 'x86_64')
+url="https://llvm.org"
+license=('custom:Apache 2.0 with LLVM Exception')
+depends=('clang')
+makedepends=('ninja')
```

Comment:
```suggestion
makedepends=('cmake' 'ninja')
```

Reply:
I have it committed locally, just waiting on `-DLLVM_LINK_LLVM_DYLIB=ON` build to finish to see if LLD needs the recent patch. That will reduce the size.

Reply:
I would be interested to know that too.  I was thinking it would be possible for something 'invasive' like cygwin/msys to implement something different for global destructors in DLLs, but I don't know if they needed to in order to get sufficiently posix-like behavior for their purposes.

Reply:
FWIW I've used build with `-DLLVM_LINK_LLVM_DYLIB=ON` for some tests and LLD haven't deadlocked even once.